Snippets/ASB/ASB_7/MultipleNamespaces.cs

```diff
@@ -0,0 +1,80 @@
+                    .AddNamespace("namespace2", "Endpoint=sb://namespace2.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("namespace3", "Endpoint=sb://namespace3.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]");
+
+            #endregion
+        }
+
+        public void FailOverNamespacePartitioning(EndpointConfiguration endpointConfiguration)
+        {
+            #region fail_over_partitioning_strategy
+
+            endpointConfiguration.UseTransport<AzureServiceBusTransport>()
+                .NamespacePartitioning().UseStrategy<FailOverNamespacePartitioning>()
+                    .AddNamespace("primary", "Endpoint=sb://primary.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("secondary", "Endpoint=sb://secondary.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]");
+
+            #endregion
+        }
+
+        public void ReplicatedNamespacePartitioning(EndpointConfiguration endpointConfiguration)
+        {
+            #region replicated_partitioning_strategy
+
+            endpointConfiguration.UseTransport<AzureServiceBusTransport>()
+                .NamespacePartitioning().UseStrategy<ReplicatedNamespacePartitioning>()
+                    .AddNamespace("namespace1", "Endpoint=sb://namespace1.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("namespace2", "Endpoint=sb://namespace2.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("namespace3", "Endpoint=sb://namespace3.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]");
+
+            #endregion
+        }
+
+        public void ShardedNamespacePartitioning(EndpointConfiguration endpointConfiguration)
+        {
+            #region sharded_partitioning_strategy_configuration
+
+            endpointConfiguration.UseTransport<AzureServiceBusTransport>()
+                .NamespacePartitioning().UseStrategy<ShardedNamespacePartitioning>()
+                    .AddNamespace("namespace1", "Endpoint=sb://namespace1.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("namespace2", "Endpoint=sb://namespace2.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]")
+                    .AddNamespace("namespace3", "Endpoint=sb://namespace3.servicebus.windows.net;SharedAccessKeyName=[shared access key name];SharedAccessKey=[shared access key]");
+
+            #endregion
```

Comment:
There will be a detailed description inside the md file to describe the differences between strategies. Snippets are only to show how to configure a specific strategy